scripts/start_supervisor.py

```diff
@@ -31,11 +31,13 @@
 #  This node runs a script in the remote python interpreter of the Stack of
 #  Tasks. The name of the script is given as input to the node.
 
+from math import sin, cos
 import rospy,sys
-from std_srvs.srv import *
-from dynamic_graph_bridge.srv import *
-from dynamic_graph_bridge_msgs.srv import *
+from std_srvs.srv import Empty
+from dynamic_graph_bridge_msgs.srv import RunCommand
+from dynamic_graph_bridge_msgs.msg import Vector
 from os.path import isfile
+from hpp import Quaternion
```

Comment:
This adds a dependency to HPP. Could you add a `\todo` note stating that this is required only for quaternion to RPY conversion ?

Reply:
This conversion can be done with:
```python
import eigenpy
q = eigenpy.Quaternion()
eigenpy.toEulerAngles(q.matrix(), 2, 1, 0)
```